### PR TITLE
Fix(raw_bin): read interval shorter than sample interval

### PR DESCRIPTION
**Pull request by jusjusjus**
Addressing this issue: https://github.com/BEL-Public/mffpy/issues/79

### CODE REVIEW

File: mffpy/raw_bin_files.py [hunk 1]
```diff
@@ -166,6 +166,12 @@ def _read_blocks(self, A: int, B: int) -> np.ndarray:
             d = np.frombuffer(buf, '<f4', count=-1)
             d = d.reshape(self.num_channels, -1, order='C')
             data.append(d)
+
+        if len(data) == 0:
+            empty_data = np.array((), dtype=np.float32)
+            empty_data.shape = (self.num_channels, 0)
+            return empty_data
```
Comment (jusjusjus) [inline mffpy/raw_bin_files.py:173]: The fix

**Review by ianbrown9475 — DISMISSED** [review]
Nice work! I rigged this up with signal service and FLOW and I now get an empty response when the interval is very small, which seems to be expected. Might not be related to this, but I noticed that if I set start time == end time, I get the full range of EEG data.

**Comment by jusjusjus** [comment]
> Nice work! I rigged this up with signal service and FLOW and I now get an empty response when the interval is very small, which seems to be expected. Might not be related to this, but I noticed that if I set start time == end time, I get the full range of EEG data.

Great catch.   Not to say that it isn't still iffy, however, now passing `dt=0.0` returns no data.

**Review by ianbrown9475 — DISMISSED** [review]

**Comment by jusjusjus** [comment]
I still needed to update the CHANGELOG.

**Review by ianbrown9475 — APPROVED** [review]

**Pull request merged**